package.json

```diff
@@ -298,83 +322,123 @@
         {
           "command": "pacCLI.authPanel.copyUser",
           "when": "never"
+        },
+        {
+          "command": "pacCLI.openDocumentation",
+          "when": "!virtualWorkspace"
```

Comment:
Pulling down the branch here and testing has shown some odd behavior.  

Using your new "Run Web Extension in VS Code" launch.json option, all of these PAC commands are still enabled.   Despite that debug environment being `isWeb`, it looks like that environment is _not_ `virtualWorkspace`

Launching the web extension in the browser `npx @vscode/test-web --extensionDevelopmentPath="MyLocalPath"` makes an envrionment that is both `isWeb` and `virtualWorkspace`, and the PAC commands are correctly hidden.

This second method is obviously the one that users will hit instead of the debug environment, but it would be helpful if the debug environment was behaving closer to how the actual target environment behaves.

So we may want to switch all these `"when": "!virtualWorkspace"` to ` "when": "!isWeb"` or ` "when": "!isWeb && !virtualWorkspace"`?

Reply:
...and ignore that suggestion. 

Tried the branch out in CodeSpaces and swapping `!virtualWorkspace` to `!isWeb`, which breaks the CodeSpaces experience as that environment uses a Web frontend to an actual non-virtual workspace.

It would still be nice if the PAC commands were hidden during "Run Web Extension in VS Code" to better match the user experience, but not at the cost of breaking the CodeSpaces scenario.